spec/widgets/widget-model.spec.js

```diff
@@ -190,6 +190,21 @@ describe('widgets/widget-model', function () {
       });
     });
 
+    describe('.getAutoStyle', function () {
+      it('should not provide any info if auto-style is not enabled', function () {
+        spyOn(this.model, 'isAutoStyleEnabled').and.returnValue(false);
+        var data = this.model.getAutoStyle();
+        expect(data).toEqual({});
+      });
+
+      it('should not provide any info if autoStyler is not defined', function () {
+        spyOn(this.model, 'isAutoStyleEnabled').and.returnValue(true);
+        this.model.autoStyler = undefined;
+        var data = this.model.getAutoStyle();
+        expect(data).toEqual({});
+      });
+    });
+
```

Comment:
This spec only test one path, the one where we shouldn't provide any info.

There was no existing specs about the other case, where we should provide info, but I think this is the right moment to add them.